Makefile

```diff
@@ -116,9 +116,9 @@ run-slice:
 ifeq (,$(wildcard nodelogs))
 	mkdir nodelogs
 endif
-	$(PRIME_CMD) >> $(PRIME_LOG_FILE) 2>&1 &
-	$(REGION_CMD) >> $(REGION_LOG_FILE) 2>&1 &
-	$(ZONE_CMD) >> $(ZONE_LOG_FILE) 2>&1 &
+	@nohup $(PRIME_CMD) >> $(PRIME_LOG_FILE) 2>&1 &
```

Comment:
run slice is invalid, we need to remove it

Reply:
how so?

Reply:
Even running a slice uses make run-all

Reply:
Then why do we have 'run-all' as a make target? We should just have 'run', if there's no longer two run targets to differentiate between.